src/elements/fields/RadioButtonGroupField.tsx

```diff
@@ -63,6 +65,23 @@ function RadioButtonGroupField({
     }));
   }
 
+  // iOS devices do not scroll to focused radio buttons
+  // we manually scroll to the focused radio button to
+  // handle form errors and provide the same experience other devices have
+  const scrollOnFocus = (event: FocusEvent<HTMLInputElement, Element>) => {
```

Comment:
since all of the scrollOnFocus functions are mostly the same, can we make it a browser utility